### PR TITLE
Add mkdocs 

### PR DESCRIPTION
**Pull request by sidhulyalkar**

### CODE REVIEW

File: docs/src/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Element Session features a DataJoint pipeline allowing for a standard approach for session
+level organization. The Element is composed of schemas for 
+
+
+runs DeepLabCut which uses image recognition machine learning models
+to generate animal position estimates from consumer grade video equipment.  The Element
+is composed of two schemas for storing data and running analysis:
+- `train` - Manages model training
+- `model` - Manages models and launches pose
+estimation
```
Comment (kabilar) [inline docs/src/index.md:17]:
```suggestion
```

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @sidhulyalkar.  Please update the `index` and `concepts` pages to reflect the latest structure and narrative from [element-deeplabcut](https://datajoint.com/docs/elements/element-deeplabcut/0.2/).

**Comment by CBroz1** [comment]
Please add [automated versioning to the citation page](https://github.com/datajoint/element-deeplabcut/commit/352c44f98325cb79a249026d84310bff8417e7e2)

**Comment by CBroz1** [comment]
replaced by #27

**Pull request closed without merging**